common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1315,6 +1315,8 @@ public final class AdminConstants {
     public static final String A_VOLUME_S3 = "S3";
     public static final String A_VOLUME_OPEN_IO = "OPENIO";
     public static final String A_VOLUME_STORE_MANAGER_CLASS = "storeManagerClass";
+    public static final String A_VOLUME_BUCKET_ID = "bucketId";
```

Comment:
is It the correct parameter name? I believe for external info it's `globalBucketConfigId` for CLI it was `bucketID` please confirm.

Reply:
It doesn't matter as we take input from user by "--bid" parameter. This shall make only difference while printing the output

Also I check previous value, it is unchanged 
private static final String A_BUCKET_ID = "bucketId";  [Old param value]

Reply:
is that mean we are different param names in SOAP and CLI? is it same for other params as well?

Reply:
Yes in SOAP API we are using A_VOLUME_GLB_BUCKET_CONFIG_ID = "globalBucketConfigId";

In CLI we are using  A_VOLUME_BUCKET_ID=bucketId

This is inconsistency but not functional issue

Reply:
@dawoodshaikh Can we update the name of this parameter as well in zmvolue help

Current param name "bucketId", new proposed name "globalBucketConfigId"

cc: @RakeshAMore

Reply:
Reply to :->
what about my second que "is it the same for other params as well?" and what are the efforts for removing this inconsistency about bucketed param?

This parameter is only inconsistent. I have already removed inconsistency for "**A_VOLUME_COMPRESS_BLOBS**" and "**A_VOLUME_USE_IN_FREQ_ACCESS_THRESHOLD**". Other parameters seems consistent.

It will cost 30 minutes to do and test it

Reply:
@inpati @RakeshAMore As discussed, lets change it to globalBucketConfigId and short option to gbid

Reply:
Doe, fixed